scst-sign/configuring.md

```diff
@@ -0,0 +1,244 @@
+# Configuring Supply Chain Security Tools - Sign
+
+This component requires extra configuration steps to start verifying your
+container images properly.
+
+## Create a `ClusterImagePolicy` Resource
+The cluster image policy is a custom resource containing the following information:
+- A list of namespaces where this policy will not be enforced.
```

Comment:
suggestion: `verification.exclude.resources.namespaces: A list ...`

Reply:
Does this change warrants changing `following information` for `following keys`?

Reply:
You are describing the properties of the CR, so it makes sense to refer them here.